### ISSUE TITLE
Add a place for staging of proposed ava's that are up for going up for public review.

### ISSUE THREAD

**Issue by danshome** [UCDavisLibrary/ava#916]
It would be nice to have a place for uploading geojson files for proposed AVA's that are going up for public review.

**Comment by danshome** [comment]
For example, add an avas_proposed directory or something similar where petitioners can upload geojson files for ava boundaries they have submitted to TTB for review; this could also act as a single place for TTB to pull geojson files for proposed ava's and allow for TTB to make any adjustments they want to suggest without having to pass files back and forth between the petitioner and TTB.

**Comment by MicheleTobias** [comment]
That's an interesting idea!  Do you think the petitioners would have the skills for that?  I should note that the TTB has it's own dataset that is separate and has a different set of methods from the UC Davis data.

**Comment by danshome** [comment]
I'm one, and I have the skills.  :)   It would be nice if this project was better organized for build automation. Who would be the best person to talk to about some of the R scripts that have been built better to understand some of their use cases? For example, AVA counties from shapefile.R.  What's the use case for that script?  It seems like it might be used to update the county attribute in an AVA geojson manually based on the county shape file downloaded from data.gov, but it's hard to tell based on the description in the script if that's the use case.

**Comment by MicheleTobias** [comment]
I suspect you are the exception! But that's really great to hear that you've mapped out your proposed AVA. So many of the descriptions are challenging to interpret because I think winemakers have winemaking skills and aren't necessarily geographers.

Mostly the scripts are for our internal use and they do things like build the aggregated files so people can download all the boundaries in one geojson file instead of having to deal with 300+ files individually. So, the pertinent scripts have already been run to make the data that's available in the avas_aggregated_files and avas_by_state folders. Typically, at the top of each file, you'll find a short description of what the file does. I think we can do a better job documenting them though and removing depreciated code. 

I've thought about automating the data processing, but since the digitizing has slowed down, it hasn't been that pressing. I do it with other [DataLab](https://datalab.ucdavis.edu/) projects. We are still working on some quality control like updating the county, within, and contains columns. We'll do those with scripts. School is just starting so I'm waiting on my student crew to get settled so we can pick back up where we left off this summer.

**Comment by danshome** [comment]
So is the use case I described for AVA counties from shapefile.R what it's used for?

**Comment by danshome** [comment]
@MicheleTobias Here is a codepin of sample of an [American Viticulture Area Boundary Builder](https://codepen.io/danshome/pen/YzdjwBd) that would make it easy for people that don't know anything about GIS to create a properly formatted geojson.  I'm sure this could somehow be tied into Github to actually submit a pull request as well.
